w_common_tools/lib/src/test_fixtures/sass/_test_package_import.scss

```diff
@@ -0,0 +1,3 @@
+.package-import {
```

Comment:
Do you know why this test fixture lives in lib/src rather than in test/unit/vm/fixtures?

Reply:
I believe it's so we can test the package import functionality of the sass compiler. [Here's where a package import is used to bring in this file.](https://github.com/Workiva/w_common/blob/add_w_common_tools_package/w_common_tools/test/unit/vm/fixtures/sass/test.scss#L5)